rear_rider_device/test_lidar_proc.py

```diff
@@ -0,0 +1,31 @@
+import asyncio
+from pkgutil import get_data
+from sys import stdout
+from ipc.parent_process import ParentProcess
+# import rear_rider_sensors.lidar as lidar
+
+class LidarParentProcess(ParentProcess):
```

Comment:
Could you rename this to `TestLidarParentProcess`? This is to differentiate it from the actual `LidarParentProcess`.

Reply:
Using the same method as `test_lidar_child_process(LidarChildProcess)`, could you also do the same here (`TestLidarParentProcess(LidarParentProcess)`) so that the only thing in this class is the overridden method `on_get_data`. It is a cosmetic change, but will make this file more pleasant :smile:.